CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 Changelog
 =========
 
+0.7.1 (2025-12-09)
```

Comment:
I guess we use a patch version because it's a bug fix right?

Reply:
Yeah, there were no other changes done after the 0.7.0 release so I guess we can make this a bug fix release.